Assets/MRTK/SDK/Experimental/PulseShader/Scripts/HideOnDevice.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
```

Comment:
nit: missing license header
```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
﻿using System.Collections;
```